lib/config.js

```diff
@@ -19,7 +19,7 @@ var findFile = function findFile (configPath, filename) {
 
   configPath = configPath || path.join(process.cwd(), filename);
 
-  if (fs.existsSync(configPath)) {
+  if (fs.existsSync(configPath) && fs.statSync(configPath).isFile()) {
```

Comment:
Checks if it's a file. My tests add a `.sass-lint.yml` directory and were failing because of this.
I think it's a good sanity check anyway.